app/src/main/java/com/kickstarter/ui/extensions/ActivityExt.kt

```diff
@@ -192,3 +196,13 @@ fun Activity.startPreLaunchProjectActivity(project: Project, previousScreen: Str
     startActivity(intent)
     TransitionUtils.transition(this, TransitionUtils.slideInFromRight())
 }
+
+fun Activity.startDisclaimerActivity(disclaimerItem: DisclaimerItems) {
+    val intent = when (disclaimerItem) {
+        DisclaimerItems.TERMS -> Intent(this, HelpActivity.Terms::class.java)
+        DisclaimerItems.PRIVACY -> Intent(this, HelpActivity.Privacy::class.java)
+        DisclaimerItems.COOKIES -> Intent(this, HelpActivity.CookiePolicy::class.java)
+        DisclaimerItems.HELP -> Intent(Intent.ACTION_VIEW, Uri.parse(Secrets.HelpCenter.ENDPOINT))
+    }
+    startActivity(intent)
+}
```

Comment:
Love it, ill use this is my stuff too!

Reply:
We might actually be able to use this as default values in the drop-down menu composable